### ISSUE TITLE
Annotation review and side-by-side comparison (to manually review the annotations from multiple annotators labeled same doc for extensive coverage

### ISSUE THREAD

**Issue by khyurri** [epam/badgerdoc#58]
## General algorithm

When validator proceed validation task in case of extension coverage, so validator requires see multiple revisions in the same screen in different frames. 

In validators UI makes request to tasks by document and job_id. Then UI requests latest revisions for selected pages from task. In case of extensive coverage validation, UI must get latest versions by each annotator (we have already this endpoint). 

## Back-end:
- [x] Check if this endpoint still working: http://dev2.badgerdoc.com/api/v1/annotation/docs#/Annotation/get_latest_revision_by_user_annotation__job_id___file_id__latest_by_user_get
- [x] Add validation type endpoint (or check if it's exists) 
http://dev1.badgerdoc.com/api/v1/annotation/tasks/taskId
- [x] Add batch endpoint to service-convert, which will do:
        - Uploads documents to assets http://dev2.badgerdoc.com/api/v1/assets/docs#/files/uploads_files_into_minio_bucket_files_post
       -  Creates jobs with name (required field) http://dev2.badgerdoc.com/api/v1/jobs/docs#/default/create_job_jobs_create_job_post
       -  Link to category and to taxonomy (upload api already exists) http://dev2.badgerdoc.com/api/v1/taxonomy/docs#/Taxonomy/associate_taxonomy_to_category_taxonomy_link_category_post
       -  Wait until tasks created 
       http://dev2.badgerdoc.com/api/v1/jobs/docs#/default/get_jobs_progress_jobs_progress_post
       -  Commit revison by job_id file_id http://dev2.badgerdoc.com/api/v1/annotation/docs#/Annotation/post_annotation_by_pipeline_annotation__job_id___file_id__post
       -  Start the job http://dev2.badgerdoc.com/api/v1/jobs/docs#/default/run_job_start__job_id__post
- [x] Add field `original_annotation_id`  to annotation POST endpoint (create new annotation)

## Design and Front-end

We need to show multiple frames/windows from every document to show different annotations
By default validator see N + 1 windows. We need to clarify, if validators window must be prefilled. Currently, validators window is empty. 
Add functionality to select correct annotation. When validator clicks on any bbox or label in any frames, this bbox copied to validators window. Validator can edit annotations only in own frame. Ctrl + Z must work, if validator miss clicked. 
Scroll must be the same for all annotations (frames) 

Validators stores own revision, compiling from other annotations

**Comment by khyurri** [comment]
~ 2w development

**Comment by thinklab** [comment]
```
curl -X 'POST' \
  'http://0.0.0.0:8080/batch_upload?category_id=test' \
  -H 'accept: application/json' \
  -H 'X-Current-Tenant: test' \
  -H 'Content-Type: multipart/form-data' \
  -F 'jobs={
  "name": "string",
  "type": "ExtractionJob",
  "files": [
    0
  ],
  "datasets": [
    0
  ],
  "is_draft": false,
  "is_auto_distribution": false,
  "validation_type": "cross",
  "annotators": [
    "string"
  ],
  "owners": [
    "string"
  ],
  "categories": [
    "string"
  ],
  "deadline": "2022-12-13T09:16:43.806Z",
  "validators": [
    "string"
  ],
  "extensive_coverage": 0,
  "pipeline_name": "string",
  "pipeline_version": "string",
  "start_manual_job_automatically": true,
  "import_source": "string",
  "import_format": "string"
}' \
  -F 'taxonomy_link_params={
  "id": "123abc"
}' \
  -F 'files=@samlpe1.txt;type=text/plain' \
  -F 'files=@samlpe2.txt;type=text/plain'
```

**Comment by khyurri** [comment]
Released in 1.2.4
